### ISSUE TITLE
Drying basin disappearing when filled with lava

### ISSUE THREAD

**Issue by wizard1399** [CyclopsMC/IntegratedDynamics#543]
#### Issue type:

_Only keep the issue types that are applicable._

- :bug: Bug

____
#### Short description:
Drying basin disappears when filled with lava. I was in creative mode and placed a bucket of lava into the drying basin. Shortly after the basin disappears from the world. 

#### Expected behaviour:
Drying basin to stay in world with lava in it. I have a craft tweaker script that relies on using lava with the drying bin.

#### Actual behaviour:
"Poof" the bin is gone.

#### Steps to reproduce the problem:

1. Creative mode
2. Place down drying basin
3. Place a bucket of lava into basin
4. After a second or two it should vanish.

____
#### Versions:

_
- This mod: 1.10.2-0.7.14
- CyclopsCore: 1.9.4-0.10.10
- Minecraft: 1.10.2
- Forge: 1.10.2-12.18.3.2422

#### Log file:
Null

**Comment by rubensworks** [comment]
That's not a bug, it's a feature!

Regular drying basins will burn down if they contain hot fluids, such as lava.

I could add a config option to disable this behaviour though...

**Comment by wizard1399** [comment]
Could you add the config to the 1.10.2 version? The minetweaker script that lead to this discovery has to do with thaumcraft and getting aspect crystals in a void world. Azanor has said that TC will be dev'ed in 1.10.2 to work most of the bugs out before moving on to 1.12.

TLDR: adding the config to 1.12 will not solve the problem for me or the magic based skyblock pack I am developing. I know this would be back tracking for you, so I would like to thank you profusely in advance.

**Comment by rubensworks** [comment]
Sorry, 1.10.2 is not maintained anymore unfortunately, I simply do not have the time for it.

**Comment by rubensworks** [comment]
@wizard1399 You're welcome to fork the 1.10.2 version and build a custom version by changing the following field yourself though: https://github.com/CyclopsMC/IntegratedDynamics/blob/master-1.12/src/main/java/org/cyclops/integrateddynamics/tileentity/TileDryingBasin.java#L33

<blockquote><img src="https://avatars2.githubusercontent.com/u/11808375?s=400&v=4" width="48" align="right"><div><img src="https://assets-cdn.github.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/CyclopsMC/IntegratedDynamics">CyclopsMC/IntegratedDynamics</a></strong></div><div>IntegratedDynamics - A Minecraft mod to take full and automated control of your appliances.</div></blockquote>

**Comment by wizard1399** [comment]
Not asking to maintain, just a quick fix. Possible comment out a few lines of code responsible for the burning down feature, slap a hotfix label on it and release it.

**Comment by ouroborus** [comment]
Maybe a fire animation if they're going to burn down? Seems like currently there's no clue as to what happened so you can't tell if this was intended or not.

**Comment by rubensworks** [comment]
It spawns a fire block after burning down.

**Comment by 7eggert** [comment]
It should spawn a lava block?
